examples/tutorials/advanced/date_time_charts.py

```diff
@@ -189,13 +189,13 @@
 # Using :class:`xarray.DataArray`
 # -------------------------------
 #
-# In this example, instead of using a :func:`pandas.date_range`, ``x`` is
+# In this example, instead of using :func:`pandas.date_range`, ``x`` is
 # initialized as a list of :class:`xarray.DataArray` objects. This object
```

Comment:
This formulation is not fully correct in my opinion. The individual dates are initialized as list within an xarray.DataArray object and not as a list of xarray.DataArray objects.